tests/components/blue_current/test_init.py

```diff
@@ -174,12 +188,29 @@ async def test_reconnect(hass: HomeAssistant) -> None:
         )
 
         connector = Connector(hass, config_entry, Client)
-        await connector.reconnect()
+
+        with patch("bluecurrent_api.Client.connect", side_effect=WebsocketError):
```

Comment:
Can we patch this at the place where the Client is used?

Reply:
Do you mean i should patch `Connector.connect` instead? (which calls Client.connect)

Reply:
The whole "homeassistant.components.bluecurrent.something". We prefer those patches (and I probably missed this last time)